phoenix-core/src/main/java/org/apache/phoenix/index/IndexMaintainer.java

```diff
@@ -892,6 +912,13 @@ public void readFields(DataInput input) throws IOException {
         int encodedEstimatedIndexRowKeyBytesAndImmutableRows = WritableUtils.readVInt(input);
         this.immutableRows = encodedEstimatedIndexRowKeyBytesAndImmutableRows < 0;
         this.estimatedIndexRowKeyBytes = Math.abs(encodedEstimatedIndexRowKeyBytesAndImmutableRows);
+        int numIndexedExpressions = WritableUtils.readVInt(input);
```

Comment:
Is it ok to serialize/deserialize indexedExpressions this way or will this break compatiblilty?